scripts/tasks/md-fixers.js

```diff
@@ -72,6 +72,19 @@ const fiddleTransformer = (line) => {
   }
 };
 
+/**
+ * Crowdin translations put markdown content right
+ * after HTML comments and thus breaking Docusaurus
+ * parse engine. We need to add a new EOL after `-->`
+ * is found.
+ */
+const newLineOnHTMLComment = (line) => {
+  if (line.includes('-->')) {
+    return line.replace('-->', '-->\n');
```

Comment:
Are there any edge cases that can bite us here -- 

- is there ever more than one html comment per line?

- will there ever already be a newline after the html comment? and if we make that a double newline, does that cause any problems with Docusaurus?

Reply:
Good questions

> is there ever more than one html comment per line?

I've only seen an HTML comment in one of our markdown docs and unfortunately (and we cannot remove it for now). It was only one comment in one line

> will there ever already be a newline after the html comment? and if we make that a double newline, does that cause any problems with Docusaurus?

Double new lines are not a problem for Docusaurus. Crowdin already returns markdown that's slightly different than the original with extra lines.